### ISSUE TITLE
Support latest Ember?

### ISSUE THREAD

**Issue by flyswatter** [kumavis/ember-vacuum-tube#2]
Is this deprecated?  The latest ember (1.13.2) has deprecated `computed.reduce` and wants normal array methods instead.

**Comment by flyswatter** [comment]
I ask if it's deprecated because you make it sound like it is in the other issue.

**Comment by kumavis** [comment]
yeah i have no intention of supporting this. going to unpublish this now -- are you using it? happy to transfer ownership

**Comment by flyswatter** [comment]
I have now moved off it!

- Dan
